lib/registrant.rb

```diff
@@ -0,0 +1,26 @@
+class Registrant 
+    attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here, you have exposed a method of .permit that reads back the value of the @permit attribute, but you have another method .permit? that does that same thing. We only want one, the .permit? one so we don't want .permit in the attr_reader.